plugin/storage/es/spanstore/reader.go

```diff
@@ -441,10 +454,12 @@ func (s *SpanReader) buildOperationNameQuery(operationName string) elastic.Query
 }
 
 func (s *SpanReader) buildTagQuery(k string, v string) elastic.Query {
-	queries := make([]elastic.Query, len(tagFieldList))
-	for i := range queries {
-		queries[i] = s.buildNestedQuery(tagFieldList[i], k, v)
+	queries := make([]elastic.Query, len(objectTagFieldList)+1)
+	kd := strings.Replace(k, ".", ":", -1)
```

Comment:
This replace is duplicated in many places. Any way it can be extracted as a helper method with a nice name?